src/opensearch_helper_functions.py

```diff
@@ -139,16 +139,19 @@ def execute_opensearch_query(query_against, request, index, es_url, query=None,
         # The use of json parameter converts python dict to json string and adds content-type: application/json automatically
     else:
         json_data = query
-
+    if general_index and 'log_query' in general_index and general_index.get('log_query') is True:
+        if query_against in ['_search']:
+            logger.info(f"Query: {json_data}")
```

Comment:
Add the target real index (not the `general_index`) to the log will be useful for filtering.
```
logger.info(f"Query against {the-target-real-index}: {json_data}")
```